src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,32 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
+type Props = {
+  todo: Todo,
+  selectTodo: (id:number) => void,
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, selectTodo }) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  const loadUsers = async () => {
```

Comment:
Consider renaming `loadUsers` to `loadUser` since only one user is being loaded. 🧐